mypyc/test/test_run.py

```diff
@@ -172,12 +172,10 @@ def run_case_inner(self, testcase: DataDrivenTestCase) -> None:
             # new by distutils, shift the mtime of all of the
             # generated artifacts back by a second.
             fudge_dir_mtimes(WORKDIR, -1)
```

Comment:
If we're just doing `time.sleep()` on all platforms now, maybe we should just delete this line (and adjust the comment above it)? Or comment it out?

```suggestion
```

Reply:
There is no harm in doing both. I would just keep it as is, and hope someone will find a better solution soon.